packages/frontend/components/api/RequestBuilder.tsx

```diff
@@ -0,0 +1,191 @@
+import React from 'react'
+import {
+  Box,
+  Button,
+  HStack,
+  Input,
+  Text,
+  Tooltip,
+  FormControl,
+  FormLabel,
+  FormErrorMessage,
+} from '@chakra-ui/react'
+import { BigNumber } from 'ethers'
+import { formatFixed } from '@ethersproject/bignumber'
+import { useContractFunction, TransactionState } from '@usedapp/core'
+import { ContractId } from '../../conf/config'
+import { Error } from '../../components/Error'
+import { useContract } from '../../hooks/useContract'
+// @ts-ignore
+import { APIRequestBuilder } from 'types/typechain'
+
+const defaultMultiplier = '1000000000000000000'
```

Comment:
```suggestion
const DEFAULT_MULTIPLIER = '1000000000000000000'
```

Please use this style for the constants.